src/settings/Models/FinePolicy/FinePolicy.test.js

```diff
@@ -0,0 +1,122 @@
+import FinePolicy from './FinePolicy';
+import { OverdueFine, Metadata } from '../common';
+
+describe('FinePolicy', () => {
+  const basicPolicy = {
+    id: 'id',
+    name: 'name',
+    description: 'description',
+    countClosed: 0,
+    maxOverdueFine: '0.00',
+    forgiveOverdueFine: 1,
+    gracePeriodRecall: 5,
+    maxOverdueRecallFine: '5.00',
+  };
+
+  const policy = {
+    ...basicPolicy,
+    overdueFine: {},
+    overdueRecallFine: {
+      quantity: 1,
+      intervalId: 'id',
+    },
+  };
+
+  const finePolicyInstance = new FinePolicy(policy);
+
+  it('"FinePolicy" should have correct properties', () => {
+    expect(finePolicyInstance).toEqual(expect.objectContaining(basicPolicy));
+  });
+
+  it('"FinePolicy" should have correct properties if "policy" is not provided', () => {
+    const finePolicy = new FinePolicy();
+
+    expect(finePolicy).toEqual(expect.objectContaining({
+      id: undefined,
+      name: undefined,
+      description: undefined,
+      countClosed: undefined,
+      maxOverdueFine: undefined,
+      forgiveOverdueFine: undefined,
+      gracePeriodRecall: undefined,
+      maxOverdueRecallFine: undefined,
+    }));
+  });
+
+  it('"overdueFine" property should be instance of "OverdueFine"', () => {
```

Comment:
I would suggest to match fields data too. Becaus if someone will drop props passing to the `new OverdueFine()` this test will not fail. We can do mock for class, or in this case, as this class not so complex, we can check it as it is. What do you think?

Reply:
I am not sure that we need to check it because it depends on properties of `FinePolicy` component and that properies can be empty.
![image](https://user-images.githubusercontent.com/42437614/170449316-6a95de29-413d-4f5a-a7eb-c501a006781b.png)
 Also, each class like `new OverdueFine()` handle empty properties

Reply:
Discussed